src/js/helpers/reviewHelper.js

```diff
@@ -172,7 +172,7 @@ const getFileStates = (status) => {
 		else {
 			let count = 0;
 			item.error_data.forEach((error) => {
-				count += parseInt(error.occurrences);
+				count += parseInt(error.occurrences, 10);
```

Comment:
Good question! Adding the `base` parameter to `parseInt` is just a sanity check. We almost never want to parse an integer in a non-10 base (and when we do, we should be explicit that that's the intent), so one of the [linting rules](http://eslint.org/docs/rules/radix) checks for this. I think that's a reasonable standard, but would be happy to revert if there's contention.